serverOptions.go

```diff
@@ -32,6 +32,9 @@ type ServerOptions struct {
 	Heartbeat             OptionValue
 	HeartbeatTimeout      time.Duration
 	HeartbeatInterval     time.Duration
+	TLS                   OptionValue
```

Comment:
TLS configuration should not be part of the webwire server options, because it's not part of webwire. TLS is used by the HTTP server that's hosting the webwire server, thus the certificate and key file paths should be passed to a [new server constructor function](https://github.com/qbeon/webwire-go/pull/34#issuecomment-430401259).